Mimir.Worker/Handler/BaseDiffHandler.cs

```diff
@@ -69,9 +69,12 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         }
     }
 
-    protected virtual async Task<(long, long, long, long)> CalculateCurrentAndTargetIndexes(
-        CancellationToken stoppingToken
-    )
+    protected virtual async Task<(
+        long CurrentBaseIndex,
+        long CurrentTargetIndex,
+        long CurrentIndexOnChain,
+        long IndexDifference
```

Comment:
Can you fix these values name as camel case?

Reply:
Hmm, C# doesn't have naming-convention for named-tuple. We should make our own convention for named-tuple case. @planetarium/9c-dx 

https://grep.app/search?q=public%20(int&filter%5Blang%5D%5B0%5D=C%23

Reply:
Or if we assume the return-type as public type and the names in the tuple as public fields, we should use PascalCase.

https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#pascal-case

> When naming public members of types, such as fields, properties, events, use pascal casing

Reply:
I recommend using PascalCase for naming, as it aligns with how non-named tuples are accessed, e.g., tuple.Item1. This consistency serves as a helpful guideline and ensures better readability and uniformity in our code. Let's follow this convention! :D